src/libs/OptionsListUtils.js

```diff
@@ -770,6 +807,15 @@ function getCategoryListSections(categories, recentlyUsedCategories, selectedOpt
         indexOffset += selectedOptions.length;
     }
 
+    const selectedOptionNames = _.map(selectedOptions, (selectedOption) => selectedOption.name);
+    const filteredRecentlyUsedCategories = _.chain(recentlyUsedCategories)
+        .map((categoryName) => ({
+            name: categoryName,
+            enabled: lodashGet(categories, `${categoryName}.enabled`, false),
+        }))
+        .filter((category) => !_.includes(selectedOptionNames, category) && category.enabled)
```

Comment:
I think filtering before mapping is better, it will result in less elements to map over